File: js/src/record/helpers/getBtcRecord.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.getBtcRecord = void 0;
const record_1 = require("../../types/record");
const getRecord_1 = require("../getRecord");
/**
 * This function can be used to retrieve the BTC record of a domain name
 * @param connection The Solana RPC connection object
 * @param domain The .sol domain name
 * @returns
 */
const getBtcRecord = (connection, domain) => {
    return (0, getRecord_1.getRecord)(connection, domain, record_1.Record.BTC, true);
};
exports.getBtcRecord = getBtcRecord;

File: python/record/helpers/get_btc_record.py
from solana.rpc.async_api import AsyncClient

from record.get_record import get_record
from types.record import Record


def get_btc_record(connection: AsyncClient, domain: str):
    """
    This function can be used to retrieve the BTC record of a domain name
    :param connection: The Solana RPC connection object
    :param domain: The .sol domain name
    :returns:
    """
    return get_record(connection, domain, Record.BTC, True)